generator/src/googleapis/codegen/languages/java/2.0.0/features.json

```diff
@@ -3,7 +3,7 @@
   "description": "Java libraries for Google APIs.",
   "generator": "java1_15",
   "releaseVersion": "2.0.0",
-  "baseClientLibrary": "2.2.0",
+  "baseClientLibrary": "2.3.0",
```

Comment:
I think this will be `2.4.0` after the the [PR](https://github.com/googleapis/google-api-java-client/pull/2435) in `google-api-client-service` is merged and released.

Reply:
Ah yep. I'll update it to `2.4.0` when it's released. Thanks!

Reply:
@lqiu96 The `v2.4.0` is released. Could you please merge this PR? We are waiting on this change to add the TPC support to our Cloud SQL Java Connector. Thanks!

Reply:
I'll proceed with the release today. The new clients should be generated tonight.